### DISCUSSION TITLE
CodeProject.AI- because it is an AI, does it have the ability to learn on its own?

### DISCUSSION THREAD

**Discussion by sanrafa2024** [codeproject/CodeProject.AI-Server#248] [Q&A]
tenho uma duvida por ser uma IA ela tem a capacidade de aprender sozinha?
I have a question because it is an AI, does it have the ability to learn on its own?

**Comment by VideoDad** [comment]
I would say "no". For example, the vision detection modules (YOLO = You Only Look Once) require a supervised learning algorithm, meaning it requires labeled data to train and learn how to identify objects in images, making it incapable of learning independently or "on its own.".
